handler/OpenIDLoginHandler.inc.php

```diff
@@ -176,6 +177,7 @@ function signOut($args, $request)
 			$router = $request->getRouter();
 			$lastProvider = $request->getUser()->getSetting('openid::lastProvider');
 			$context = Application::get()->getRequest()->getContext();
```

Comment:
Just a small note here: The code already has access to `$request`. Wouldn't it be more consistent to use `$context = $request->getContext();` as we do in other parts of the code, instead of `$context = Application::get()->getRequest()->getContext();`, for the sake of uniformity?